### ISSUE TITLE
[Chrome] Last header cell has incorrect width after resize

### ISSUE THREAD

**Issue by Mari4ka89** [AllenFang/react-bootstrap-table#965]
Hello,

We have found an issue with incorrect sizing for the last table header cell in Chrome.

In BootstrapTable.js: 

_adjustHeaderWidth() {

 const isScroll = headerContainer.offsetWidth !== tbody.parentNode.offsetWidth; // Here Chrome returns true while FF returns false (FF is right)
...
 const scrollBarWidth = isScroll ? Util.getScrollBarWidth() : 0;
}

So Chrome rounds offsetWidth differently. Could you please implement cross-browser solution for rounding width. For reference: http://stackoverflow.com/questions/24104905/offsetwidth-not-same-in-different-browsers

**Comment by AllenFang** [comment]
@Mari4ka89, I use the chrome@53 & 55, and setup a three column table with width configuration on last column, but I cant reproduce this issue(BTW, I also give a height for making a vertical scrollbar, but it still  fine)

I'm wondering what's version of chrome you used. Anyway, there's any thing I missing?

**Comment by mbrainiac** [comment]
@AllenFang 
Use browser with high resolution (or you can zoom in/out to see the problem).

```
const isScroll = headerContainer.offsetWidth !== tbody.parentNode.offsetWidth;
```

should be
```
const isScroll = headerContainer.offsetWidth !== tbody.parentNode.parentNode.offsetWidth;
```

You compare width of div element and table element so the size may not be correct.

**Comment by Mari4ka89** [comment]
@AllenFang I use Chrome 55 on Linux. We don't want to specify columns width as we need dynamic size. We don't use width: 100%, it is less. Our container size can potentially be with fractional digits. For example please visit http://jsfiddle.net/f35j2/show/ in Chrome and FF.
Possible solution is to use not offsetWidth but element.getBoundingClientRect().width.

**Comment by AllenFang** [comment]
@mbrainiac, 
if use 
```const isScroll = headerContainer.offsetWidth !== tbody.parentNode.parentNode.offsetWidth;``` is supposed to be `false` always.

This line is try to find out whether there's a vertical scroll bar exist, so you should use the table body instead of div to compare their width. 

And I thought @Mari4ka89's feedback is most closed to the root cause, but I still can't reproduce this issue even I have already used newest Chrome. Anyway, I'll have some survey on `getBoundingClientRect`. 

Thanks all your feedback.

**Comment by AllenFang** [comment]
@Mari4ka89 and @mbrainiac, could you please test a beta version ? `react-bootstrap-table@2.10.0-beta.1`  try ```npm install react-bootstrap-table@2.10.0-beta.1```

Let me know if the problem still there or not. thanks

**Comment by neospyk** [comment]
Hi @AllenFang, I had the same issue with @Mari4ka89  and in new 2.10.0-beta.1 looks good, last header cell has correct resizing

**Comment by AllenFang** [comment]
@neospyk, sound great ❤️

**Comment by Mari4ka89** [comment]
Hello @AllenFang, I have tested fix and it works perfect. Thank you.

**Comment by AllenFang** [comment]
@Mari4ka89, sound great ❤️

**Comment by AllenFang** [comment]
Released on `v2.10.0`
